src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
```

Comment:
Don't use two-dimension arrays and HashMap(or any other Map). They may simplify the work for you, but for now you will get more value from solving this task if you complete it without using these data structures. Let's do it manually.

Reply:
Fix comment